### DISCUSSION TITLE
Mode 3 grain cart pipe position

### DISCUSSION THREAD

**Discussion by kidshizz** [Courseplay/courseplay#7678] [Q&A] [answered]
Hi, love the mod. I'm on fs19 courseplay version 6.5.0.0, and am wondering if there is a way to save the pipe position on the grain cart, or any tips? The grain cart driver folds the pipe after dumping in truck, and then when comes back to dump again, unfolds pipe all the way. That causes a few different problems, the main one being that it won't dump due to the pipe being buried into the trailer which don't allow the grain cart to dump. I don't know if it throws the trigger off or what, but regardless, I'm wondering if there is a fix for this issue? I really love using this mode.

**Comment by Tensuko** [comment, marked as answer]
That is inside 6.5.0.0 and I remember it is on the same page where you set the unload values and stuff. Just have a look.
But it is a Giants function that it have to be folded all the way 1st, before it moves to the saved position.
Can't (and won't anymore) do something about this.

**Reply by kidshizz** [reply]
I appreciate the reply... I figured you would not be updating fs19 version of cp anymore. Anyway your answer helped. I was looking for an option about the pipe. It is in the vehicle specific settings page but called "loading position"
